spec/models/form_profile_spec.rb

```diff
@@ -34,7 +34,24 @@
       ],
       'currentlyActiveDuty' => {
         'yes' => true
-      }
+      },
+      'veteranAddress' => {
+        'street' => user.va_profile[:address][:street],
+        'city' => user.va_profile[:address][:city],
+        'state' => user.va_profile[:address][:state],
+        'country' => user.va_profile[:address][:country],
+        'postal_code' => user.va_profile[:address][:postal_code]
```

Comment:
I think this is a test, but I have a couple questions related to the code that's being tested.

The schema appears to switch between `postalCode` and `zipcode` depending on the country. Is that handled somewhere?

Also, right now we're not sending the http header that converts snake case to camel case when we make this request on the front end, so `postal_code` doesn't match the data we expect. I'm still looking into seeing if passing that header causes any problems now, but might be worth switching the case in the mapping to match the schema and the rest of the data.

Reply:
Good question. I'll look into this and follow up with you on it.

Reply:
Ok, so upon reviewing the json schemas, I could only find both `zipcode` and `postalCode` in the 1010ez form. It appears only `postalCode` is used in the education forms. I need to look into if it's being calculated anywhere for HCA but I'll work on fixing the casing for education forms.